osu.Game.Rulesets.Osu.Tests/Mods/TestSceneOsuModMuted.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using NUnit.Framework;
+using osu.Framework.Testing;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Osu.Mods;
+
+namespace osu.Game.Rulesets.Osu.Tests.Mods
+{
+    public class TestSceneOsuModMuted : OsuModTestScene
+    {
+        /// <summary>
+        /// Ensures that a final volume combo of 0 (i.e. "always muted" mode) constantly plays metronome and completely mutes track.
+        /// </summary>
+        [TestCase(0.0, 1.0)]
+        public void TestZeroFinalCombo(double expectedTrackVolume, double expectedMetronomeVolume) => CreateModTest(new ModTestData
```

Comment:
It makes little sense to have this be a parameterised test but only ever supply a single valid set of parameters - which can in addition be the *only* valid set of parameters. Why are these 0/1 values not inlined?

Reply:
Oh that was actually just a leftover from another idea I had of testing this that turned out to be invalid/incorrect now, will remove it.